site/cypress/e2e/pronouns_change.spec.js

```diff
@@ -0,0 +1,76 @@
+
+describe('Tests cases abut changing user pronouns', () => {
+
+    let oldPronouns = '';
+
+    it('Tests changing pronouns for student', () => {
+        cy.visit('/user_profile');
+        cy.login('student');
+
+        //open pronouns form
+        cy.get('#pronouns_val').click();
+        const e = cy.get('#user-pronouns-change');
```

Comment:
```suggestion
        cy.get('#user-pronouns-change').as('e');
```
[Assigning Return Values](https://docs.cypress.io/guides/references/best-practices#Assigning-Return-Values)

Reply:
Fixed according to these suggestions